api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -369,24 +369,15 @@ private void setDbWorkspaceFields(
   public ResponseEntity<EmptyResponse> deleteWorkspace(
       String workspaceNamespace, String workspaceId) {
     recordOperationTime(
-        () -> deleteWorkspaceImpl(workspaceNamespace, workspaceId), "deleteWorkspace");
+        () -> {
+          DbWorkspace dbWorkspace = workspaceService.getRequired(workspaceNamespace, workspaceId);
+          workspaceService.deleteWorkspace(dbWorkspace);
+          workspaceAuditor.fireDeleteAction(dbWorkspace);
```

Comment:
If you want, you could also inject the auditor into the service and fire from there. This is the only controller that has one I think, and that was due to how it was structured at the time. It's fine here though.

Reply:
I left it in the controller because it seemed like that was the level where all of the audit actions were being fired and it made sense to me if the intent is to track user actions